recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   - bob.extension
   - bob.blitz
   - bob.core
-  - hdf5
+  - hdf5 1.8.15*
```

Comment:
> Long story short... We really need to get our own libnetcdf ASAP.

Agreed. We also needed pinning about 6 months ago, and whilst we don't have it from conda-build, we have the next best thing - a robot! 😄 

Here is my matrix of pinned versions:

```
pinned = {'jpeg': 'jpeg 9*',
          'libtiff': 'libtiff 4*',
          'xz': 'xz 5.2.*',
          'libnetcdf': 'libnetcdf 4.3*',
          'hdf5': 'hdf5 1.8*',
          'tk': 'tk 8.5*',
          'zlib': 'zlib 1.2*',
          'readline': 'readline 6.2*',
          'openssl': 'openssl 1.0.2g',
          'libtiff': 'libtiff 4.0.*',
          'freetype': 'freetype 2.5.*',
          'libpng': 'libpng 1.6*',
          }
```

Please feel free to suggest alterations.